torch/cuda/__init__.py

```diff
@@ -539,7 +540,7 @@ def set_device(device: _device_t) -> None:
         torch._C._cuda_setDevice(device)
 
 
-def get_device_name(device: Optional[_device_t] = None) -> str:
+def get_device_name(device: Optional["Device"] = None) -> str:
```

Comment:
Torch.types Device is already an optional though...

Reply:
Yes, can linters find Optional[Optional[...]]?

Reply:
Ah yes! Will update.